docs/getting-started/install-scylla/install-on-linux.rst

```diff
@@ -0,0 +1,217 @@
+.. |UBUNTU_SCYLLADB_LIST| replace:: scylla-5.2.list
+.. |CENTOS_SCYLLADB_REPO| replace:: scylla-5.2.repo
+
+.. The |RHEL_EPEL| variable needs to be adjuster per release, depening on support for RHEL.
+.. 5.2 supports Rocky/RHEL 8
+.. When RHEL 9 is supported, add https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+.. |RHEL_EPEL| replace:: https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+======================================
+Install ScyllaDB Linux Packages
+======================================
+
+We recommend installing ScyllaDB using :doc:`ScyllaDB Web Installer for Linux </getting-started/installation-common/scylla-web-installer/>`,
+a platform-agnostic installation script, to install ScyllaDB on any supported Linux platform.
+Alternatively, you can install ScyllaDB using Linux packages. 
+
+This article will help you install ScyllaDB on Linux using platform-specific packages.
+
+Prerequisites
+----------------
+
+* Ubuntu, Debian, CentOS, or RHEL (see :doc:`OS Support by Platform and Version </getting-started/os-support>`
+  for details about supported versions and architecture)
+* Root or ``sudo`` access to the system
+* Open :ref:`ports used by ScyllaDB <networking-ports>`
+
+Additional requirements for CentOS and RHEL:
+
+* YUM package management tool
```

Comment:
`YUM` was replaced with `dnf` in the latest versions so I'll go with `yum/dnf package management tool`
(although it's part of every Centos/RH OS and it may be unnecessary to mention it)

Reply:
If it comes with every CentOS and RH version now, I'll delete the information from the requirements.